-- MySQL dump 10.13  Distrib 5.1.69, for redhat-linux-gnu (x86_64)
--
-- Host: cabot    Database: warepubs
-- ------------------------------------------------------
-- Server version	5.1.51

/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
/*!40101 SET NAMES utf8 */;
/*!40103 SET @OLD_TIME_ZONE=@@TIME_ZONE */;
/*!40103 SET TIME_ZONE='+00:00' */;
/*!40014 SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 */;
/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;

--
-- Dumping data for table `agency`
--

LOCK TABLES `agency` WRITE;
/*!40000 ALTER TABLE `agency` DISABLE KEYS */;
INSERT INTO `agency` VALUES (1,'unknown',''),(2,'NSF','http://www.nsf.gov/awardsearch/showAward?AWD_ID='),(3,'USDA',''),(4,'DOE',''),(5,'USDA ARS','');
/*!40000 ALTER TABLE `agency` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Dumping data for table `funding`
--

LOCK TABLES `funding` WRITE;
/*!40000 ALTER TABLE `funding` DISABLE KEYS */;
INSERT INTO `funding` VALUES (1,1,'unknown','',NULL),(2,2,'Gramene: A Platform for Comparative Plant Genomics','0703908','The Gramene database (http://www.gramene.org) is an online resource, jointly supported by NSF and the US Department of Agriculture\'s Agricultural Research Service. It integrates the genomic, genetic and phenotypic information in rice, maize and other cereals, thereby giving scientists and other end-users easy access to this integrated information. This project will provide for the enhancement of Gramene by incorporating biological pathway and genetic diversity information from maize, rice, wheat, sorghum, and other cereals into the resource. Comparative genomics tools will be developed, thereby allowing researchers to use knowledge gained in one plant species to identify and characterize functionally significant genes and other elements in the genomes of other species. Scientists can use the resource to make advances in our fundamental understanding of the plant processes of economic importance such as hybrid vigor, grain development, seed dormancy, drought tolerance, and resistance to diseases. In addition, the tools developed will allow for the estimation of the breeding value of individual genetic variants, thereby providing breeders with the ability to select the ideal combinations of seed stocks to create varieties that have desirable traits such as robustness, the ability to grow in marginal environments, or have high potential as a source for biofuels and other materials of high economic value. \r\n\r\nInformation resources developed, and being developed, through genomics efforts are key elements to advance our fundamental knowledge base for a future bio-based economy and to address the expected need for feeding an expanding world population. Many of the information resource are still underutilized because of the fragmentation of the datasets and the absence of tools to make meaningful connections among them. To fully unlock the potential of plant genome data, the diverse datasets must be integrated so that information is shared both within and between species. It is one of the goals of Gramene to provide that integration. Another goal of the project is to deliver the integrated dataset into the hands of plant geneticists, molecular biologists, evolutionary biologists and breeders by providing compelling, intuitive, user interfaces. Lastly, the project will reach out to students, the public, and to underrepresented minorities via a series of online tutorials and on-site workshops that involves a novel, and cost-effective, public/private partnership.\r\n\r\nAll the information resources generated by Gramene will be available to scientists, breeders, and members of the general public free of charge and without intellectual property restrictions.'),(3,2,'CPGS Oryza Genome Evolution','1026200','PI: Rod A. Wing (University of Arizona)\r\n\r\nCoPIs: Scott A. Jackson (Purdue University), Manyuan Long (University of Chicago), Carlos A. Machado (University of Maryland), and Michael J. Sanderson (University of Arizona)\r\n\r\nCollaborators: O. Panaud (University of Perpignan, France) and D. Weigel (Max Planck Institute for Developmental Biology, Germany), Doreen Ware (Cold Spring Harbor Laboratory), Qifa Zhang and Sibin Yu (Huazhong Agricultural University, China), Bin Han (National Center for Gene Research, China), Marco Wopereis (Africa Rice Center (WARDA), Benin), Mathias Lorieux (International Center for Tropical Agriculture (CIAT), Columbia, and Georgia Eizenga (DB NRRC, Germany.\r\n\r\nIntellectual merit. Asian cultivated rice (Oryza sativa) feeds more people than any other crop. As the rice-dependent population is expected to double in about 25 years, breeders are faced with the enormous task of doubling rice yields with less land, water, and fertilizers, and on poorer soils. It is therefore critical that the scientific community unites to provide the tools and biological/evolutionary insights required to meet future needs. The project\'s long-term goal is to exploit 15 million years of evolutionary diversification and adaptation that have been locked in the genomes of wild and domesticated Oryza species, to both improve cultivated rice and address fundamental questions in evolutionary and comparative genomics. To accomplish this goal the project will first generate a set of publicly available genomics resources for the genus Oryza including: i) physical maps for three wild relatives of cultivated rice and the closely related outgoup species Leersia perrieri; ii) complete sequences for the short arms of chromosome 3 of these species; iii) complete reference genome sequences for O. barthii (the wild progenitor of West African cultivated rice), and O. punctata (a wild species that will serve as the evolutionary outgroup the eight AA genome species--the species group containing O. sativa); iv) gene and small RNA expression data sets for 11 Oryza species and L. perrieri; and v) DNA polymorphism data sets across the AA genome phylogeny. The project will integrate these new resources with 5 to 8 forthcoming new Oryza reference genome sequences to create the most comprehensive within-genus comparative genomics platform for any plant system. The project will interrogate this comparative genomics platform to discover key evolutionary events and characterize the forces that led to the formation of 24 Oryza species with 10 different genome types. Specific research areas that will be addressed include: analyses of structural variation, phylogenomics, population genomics, genome evolution and the origin of new genes, and the role of transposable elements in genome evolution. Libraries, genome sequences and their annotations will be accessible via public databases and repositories, such as the AGI BAC/TEST Resource Center (http://www.genome.arizona.edu/orders/), NCBI (http://www.ncbi.nlm.nih.gov/), and Gramene (www.gramene.org).\r\n\r\nBroader impacts. This project will have broad impacts at several levels, including basic and applied research communities, postdoctoral/graduate/undergraduate and high school student training and mentoring, community outreach, and K-5 children and their families. For basic/applied research communities, the project will enhance and develop a genus-level resource as a platform for accessing new traits and alleles that is unparalleled in the plant world. This transformative platform can be used to address fundamental hypothesis-driven questions as well as provide essential tools to improve the world\'s most important food crop. The project will provide training and mentoring to postdoctoral scientists, graduate/undergraduate students and high school students interested in genome evolution, plant breeding, and careers in academic and corporate science. Finally, the project will develop a biannual \"Plant Science Family Night\" program, targeting K-5 students and families, to get children and their families excited about plants and the role plant science plays in ensuring a safe, sustainable, and secure food supply for our planet.'),(4,2,'Gene Discovery in the Cereals','1032105','PI: W. Richard McCombie (Cold Spring Harbor Laboratory) \r\n\r\nCo PI: Doreen Ware (Cold Spring Harbor Laboratory)\r\n\r\nKey Collaborator: Robert A. Martienssen (Cold Spring Harbor Laboratory)\r\n\r\nGenomic sequence data holds the promise of dramatically advancing both the understanding of basic plant science, and of catalyzing practical advances in plant breeding. While the \"complete\", contiguous, \"base perfect\" sequence of a genome is the most useful outcome of a genome project, it is not necessary for most applications. Furthermore, base perfect sequence is prohibitive in terms of both time and cost for very large genomes or large numbers of moderately sized genomes. However, recent technological advances and soon-to-be broadly available technologies provide an attractive option for plant genomics. This project seeks to use next generation sequence technologies to identify the vast majority of genes in three closely related progenitor genomes of wheat (AA, AABB, and DD genome containing species) and hexaploid wheat (Chinese spring). Each genome will be sequenced to about 50x coverage with Illumina sequencers (paired end 100 base reads) and to about 8x-10x coverage using the Pacific Bio-Sciences sequencing technology. In addition, the project will sequence each genome with long range \"strobe\" reads on the Pacific Biosciences instrument that will amount to about 2 terabases of raw sequence. The sequences will be assembled and annotated for maximum use by the broader scientific community. All raw data, assemblies, and annotations will be released to the public as soon as they are generated and quality checked and prior to any publication that might be pursued. \r\n\r\nThe broader impact of this project is to solve the data availability problem by providing about 2 terabases of raw genome sequence data to the broader cereal research community as well as the assemblies and annotation of the vast majority of genes in a large number of cereal genomes. It is expected that access to these data will radically accelerate functional genomics in wheat. All data generated in this project can be accessed through the project website, GenBank, and through the iPlant Collaborative (http://www.iplantcollaborative.org/). In addition, project updates will be coordinated with the International Wheat Genome Sequencing Consortium (IWGSC) and made available at (http://www.wheatgenome.org/Projects) and at GrainGenes (http://wheat.pw.usda.gov/GG2/index.shtml).'),(5,4,'Kbase','',''),(6,2,'The iPlant Collaborative: A Cyberinfrastructure-Centered Community for a New Plant Biology','0735191','University of Arizona: Richard Jorgensen (PI), Gregory Andrews, Kobus Barnard, Susan Brown, Vicki Chandler, Nirav Merchant, Carolyn Napoli, Sudha Ram, Steven Rounsley\r\n\r\nArizona State University: Daniel Stanzione\r\n\r\nCold Spring Harbor Laboratory: Lincoln Stein, Doreen Ware\r\n\r\nPurdue University: Rebecca Doerge, \r\n\r\nUniversity of North Carolina at Wilmington: Ann Stapleton\r\n\r\nThe iPlant Collaborative (iPC) is a new type of organization - a cyberinfrastructure collaborative for the plant sciences - that enables new conceptual advances through integrative, computational thinking (i.e. thinking at multiple levels of abstraction using a systems-level approach to problem-solving). The iPC is fluid and dynamic, utilizing new computer, computational science and cyberinfrastructure solutions to address an evolving array of grand challenges in the plant sciences. It is community-driven, involving plant biologists, computer and information scientists and engineers, as well as experts from other disciplines, all working in integrated teams. The iPC brings together strengths in plant biology, bioinformatics, statistics, computer science and high throughput computing, as well as innovative approaches to education, outreach, and the study of social networks.\r\n\r\nSeveral key principles guided the development of the iPC. Specifically, the iPC:\r\n\r\n- is a cyberinfrastructure collaborative rather than purely a cyberinfrastructure,\r\n\r\n- will enable multi-disciplinary teams to address grand challenges in plant science,\r\n\r\n- will be an entity that is by, for and of the community,\r\n\r\n- will train the next generation in computational thinking, and\r\n\r\n- is designed to be able to reinvent itself as needs and technologies change.\r\n\r\nThe driving force behind the iPC is the nature of the grand challenge questions in plant sciences, and all facets of the collaborative are organized around those selected questions. The act of selecting these questions will be community-driven, and to facilitate that, the Collaborative will host a series of workshops, each focused on a specific area of plant biology, but with participants cutting across the spectrum of the computational and biological sciences. The goal of each workshop will be to identify the grand challenge questions in that field, as well as the necessary strategies and approaches that will be needed to solve the question(s). Self-forming Grand Challenge Teams from the community will then work with iPC personnel to develop a -Discovery Environment- (DE), which will be a cyberinfrastructure for open-access research and education focused on a grand challenge question. Over time, the DEs designed for different grand challenges will overlap and coalesce into a comprehensive cyberinfrastructure for discovery and learning.\r\n\r\nThe cyberinfrastructure created by the iPC will provide the community with two main capabilities: it will provide access to world-class physical infrastructure - for example persistent storage, and compute power via local and national resources, and it will provide services that promote interactions, communications and collaborations and that advance the understanding and use of computational thinking in plant biology. Through these capabilities, the iPC will catalyze progress in targeted areas of plant biology, and more broadly advance the whole of plant science through new, creative, synthesis activities, and training the next generation of scientists in computational (and collaborative) thinking.\r\n\r\nThe broader impacts of the iPC project will not be limited merely to creating the tools for solution of currently intractable grand challenge questions, because at its core the iPC is actually a community building and educational enterprise designed to facilitate education and outreach. Grand Challenge teams and iPC staff will work together to educate students (K-12, undergraduate, and graduate, including members of underrepresented groups) through the use and development of Discovery Environments. Thus, education and outreach efforts will permeate the iPlant Collaborative.'),(7,5,'Comparative genomic analyses, bioinformatics, and resource development for cereal genomes','',''),(8,2,'Physical Mapping of the Wheat D Genome','0701916',''),(9,2,'Sequencing the Maize Genome','0527192',''),(10,2,'Molecular and Functional Diversity in the Maize Genome','0321467',''),(11,2,'YIA: Genomics of Rice, Sorghum and Maize','0333074',''),(12,2,'Collaborative Research: An Arabidopsis Polymorphism Database','0723935',''),(13,2,'High Density Scoreable Markers for Maize Trait Dissection','0638566',''),(14,2,'The Plant Ontology Consortium','0321666',''),(15,2,'Oryza Map Alignment Project','0321678',''),(16,2,'Gramene: A Platform for Comparative Cereal Genomics','0321685',''),(17,4,'Sorghum Biomass/Feedstock Genomics Research for Bioenergy','','http://genomicscience.energy.gov/research/DOEUSDA/abstracts/2006Rooney_abstract.shtml');
/*!40000 ALTER TABLE `funding` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Dumping data for table `pub`
--

LOCK TABLES `pub` WRITE;
/*!40000 ALTER TABLE `pub` DISABLE KEYS */;
INSERT INTO `pub` VALUES (1,1989,'Assessment of the number of free cysteines and isolation and identification of cystine-containing peptides from acetylcholine receptor','Biochemistry. 1989 Apr 18;28(8):3469-82','2742850','Kellaris KV, Ware DK, Smith S, Kyte J','','','',0,'','',''),(2,1992,'Primary structure, chromosomal localization, and functional expression of a voltage-gated sodium channel from human brain.','Proc Natl Acad Sci U S A. 1992 Sep 1;89(17):8220-4','1325650','Ahmed CM, Ware DH, Lee SC, Patten CD, Ferrer-Montiel AV, Schinder AF, McPherson JD, Wagner-McPherson CB, Wasmuth JJ, Evans GA, et al.','','','',0,'','',''),(3,1995,'The phenylalanine ammonia-lyase gene family in Arabidopsis thaliana','Plant Mol Biol. 1995 Jan;27(2):327-38','7888622','Wanner LA, Li G, Ware D, Somssich IE, Davis KR','','','',0,'','',''),(4,1998,'Arabidopsis Rho-related GTPases: differential gene expression in pollen and polar localization in fission yeast.','Plant Physiol. 118(2):407-17','9765526','Li H, Wu G, Ware D, Davis KR, Yang Z.','','','',0,'','',''),(5,1999,'Handling and preservation of clones in a DNA stock center','Wilson, Z. ed. A Practical Approach: Arabidopsis. Oxford Univ. Press. pp 29-50. (Book chapter)','','Scholl R, Ware D, Davis K','','','',0,'','',''),(6,2000,'Seed and molecular resources for Arabidopsis','Plant Physiol. Dec;124(4):1477-80','11115863','Scholl, RL, May ST, Ware DH','','','',0,'','',''),(7,2002,'Gramene a tool for Grass Genomics','Plant Physiol. 2002 Dec;130(4):1606-13','12481044','Ware D, Jaiswal P, Ni J, Yap I, Pan X, Clark K, Teytelman L, Schmidt S, Zhao W, Chang K, Cartinhour S, Stein L, McCouch S','','','',0,'','',''),(8,2002,'Development and mapping of 2240 new SSR markers for rice (Oryza sativa L.)','DNA Res. 2002 Dec 31;9(6):199-207','12597276','McCouch SR, Teytelman L, Xu Y, Lobos KB, Clare K, Walton M, Fu B, Maghirang R, Li Z, Xing Y, Zhang Q, Kono I, Yano M, Fjellstrom R, DeClerck G, Schneider D, Cartinhour S, Ware D, Stein L','','','',0,'','',''),(9,2002,'Development and mapping of 2240 new SSR markers for rice (Oryza sativa L.) (supplement)','DNA Res. 2002 Dec 31;9(6):257-79','12597280','McCouch SR, Teytelman L, Xu Y, Lobos KB, Clare K, Walton M, Fu B, Maghirang R, Li Z, Xing Y, Zhang Q, Kono I, Yano M, Fjellstrom R, DeClerck G, Schneider D, Cartinhour S, Ware D, Stein L','','','',0,'','',''),(10,2002,'Gramene: development and integration of trait and gene ontologies for rice','Comparative and Functional Genomics 3: 132-136','18628886','Jaiswal P, Ware D, Ni J, Chang K, Zhao W, Schmidt S, Pan X, Clark K, Teytelman L, Cartinhour S, Stein L, McCouch S','','','',0,'','',''),(11,2002,'Gramene: a resource for comparative grass genomics','Nucleic Acids Res., Jan 1;30(1):103-5.','11752266','Ware D, Jaiswal P, Ni J, Pan X, Chang K, Clark K, Teytelman L, Schmidt S, Zhao W, Cartinhour S, McCouch S, Stein L','','','',0,'','',''),(12,2003,'Maintaining Collections of Mutants for Plant Functional Genomics','Humana Press, Methods Mol Biol., 2003; 236:311-26','14501073','Scholl R, Sachs MM, Ware D','','','',0,'','',''),(13,2003,'Rice chromosome 10 reveals a high degree of heterchromatin and collinearity with cereals','Science June 300 (5629) 1566-15','','The Rice Chromosome 10 Sequencing Consortium','','','',0,'','',''),(14,2003,'Comparison of genes among cereals','Curr Opin Plant Biol 6: 121-127','12667867','Ware D, Stein L','','','',0,'','',''),(15,2004,'Identifying microRNAs in plant genomes','Proc. IEEE, CBS: 718-723','','Maher C, Timmermans M, Stein L, Ware D','/documents/CSB2004.pdf','','',0,'','',''),(16,2005,'Plant Ontology (PO): A Controlled Vocabulary of Plant Structures and Growth Stages','Comp Funct Genomics. 2005;6(7-8):388-97','18629207','Jaiswal P, Avraham S, Ilic K, Kellogg EA, McCouch S, Pujar A, Reiser L, Rhee SY, Sachs MM, Schaeffer M, Stein L, Stevens P, Vincent L, Ware D, Zapata F','PMC2447502','','',0,'','',''),(17,2005,'Toward Sequencing the Sorghum Genome','Plant Physiol. Aug; 138: 1898-1902','16172096','Sorghum Genomics Planning Workshop Participants. (2005) A U.S. National Science Foundation-Sponsored Workshop Report','','','',0,'','',''),(18,2006,'Whole-plant growth stage ontology for angiosperms and its application in plant biology','Plant Physiol. 2006 Oct;142(2):414-28. Epub 2006 Aug 11.','16905665','Pujar A, Jaiswal P, Kellogg EA, Ilic K, Vincent L, Avraham S, Stevens P, Zapata F, Reiser L, Rhee SY, Sachs MM, Schaeffer M, Stein L, Ware D, McCouch S','','','',0,'','',''),(19,2006,'Integration of hybridization-based markers (overgos) into physical maps for comparative and evolutionary explorations in the genus Oryza and in Sorghum','BMC Genomics. 2006 Aug 8;7:199.','16895597','Hass-Jacobus BL, Futrell-Griggs M, Abernathy B, Westerman R, Goicoechea JL, Stein J, Klein P, Hurwitz B, Zhou B, Rakhshan F, Sanyal A, Gill N, Lin JY, Walling JG, Luo MZ, Ammiraju JS, Kudrna D, Kim HR, Ware D, Wing RA, San Miguel P, Jackson SA','','','',0,'','',''),(20,2006,'Evolution of Arabidopsis microRNA families through duplication events','Genome Res. 2006 Apr;16(4):510-9. Epub 2006 Mar 6','16520461','Maher C, Stein L, Ware D.','','','',0,'','',''),(21,2006,'Gramene: a bird\'s eye view of cereal genomes','Nucleic Acids Res. 2006 Jan 1;34(Database issue):D717-23','16381966','Jaiswal P, Ni J, Yap I, Ware D, Spooner W, Youens-Clark K, Ren L, Liang C, Zhao W, Ratnapu K, Faga B, Canaran P, Fogleman M, Hebbard C, Avraham S, Schmidt S, Casstevens TM, Buckler ES, Stein L, McCouch S.','','','',0,'','',''),(22,2006,'Panzea: a database and resource for molecular and functional diversity in the maize genome','Nucleic Acids Res 34: D752-757','16381974','Zhao, W., P. Canaran, R. Jurkuta, T. Fulton, J. Glaubitz, E. Buckler, J. Doebley, B. Gaut, M. Goodman, J. Holland, S. Kresovich, M. McMullen, L. Stein, and D. Ware','','','',0,'','',''),(23,2006,'Look-Align: an interactive web-based multiple sequence alignment viewer with polymorphism analysis support','Bioinformatics. 2006 Feb 10; [Epub ahead of print]','16473875','Canaran P, Stein L, Ware D.','','','',0,'','',''),(24,2007,'Gramene','Methods Mol Biol. 2007;406:315-29','','Ware D.','','','',0,'','',''),(25,2007,'The Oryza Map Alignment Project (OMAP): A New Resource for Comparative Genome Studies within the Oryza','Rice Functional Genomics: Challenges, Progress and Prospects, N.M. Upadhyaya, Editor. 2007, Springer','18287700','Wing R, Kim H, Goicoechea J, Yu Y, Kudrna D, Zuccolo A, Ammiraju J, Luo M, Nelson W, Ma J, San Miguel P, Hurwitz B, Ware D, Brar D, Mackill D, Soderlund C, Stein L, Jackson S','','','',0,'','',''),(26,2007,'The Oryza map alignment project (OMAP): A new resource for comparative genomics studies within Oryza','Rice Genetics V: Proceedings of the Fifth International Rice Genetics Symposium, D.a.H. Brar, B., Editor. 2007, IRRI: Manila','','Wing R, Kim H, Goicoechea JL, Yu Y, Kudrna D, Zuccolo A, Ammiraju JS, Luo M, Nelson W, Soderlund C, San Miguel P, Gill N, Walling JG, Jackson S, Hurwitz B, Ware D, Stein L, Brar D, Mackill D','','','',0,'','',''),(27,2007,'Patterns of Selection and Tissue-Specific Expression among Maize Domestication and Crop Improvement Loci','Plant Physiol. 2007 Jul;144(3):1642-53. Epub 2007 May 11','17496114','Canaran P, Buckler ES, Glaubitz JC, Stein L, Sun Q, Zhao W, Ware D','http://www.plantphysiol.org/cgi/content/abstract/144/3/1642','','',0,'','',''),(28,2007,'The plant structure ontology, a unified vocabulary of anatomy and morphology of a flowering plant','Plant Physiol. 2007 Feb;143(2):587-99. Epub 2006 Dec 1','17142475','Ilic K, Kellogg EA, Jaiswal P, Zapata F, Stevens PF, Vincent LP, Avraham S, Reiser L, Pujar A, Sachs MM, Whitman NT, McCouch SR, Schaeffer ML, Ware DH, Stein LD, Rhee SY','','','',0,'','',''),(29,2008,'Gramene: a growing plant comparative genomics resource','Nucleic Acid Research (Database issue D947-953)','17984077','Liang C, Jaiswal P, Hebbard C, Avraham S, Buckler ES, Casstevens T, Hurwitz B, McCouch S, Ni J, Pujar A, Ravenscroft D, Ren L, Spooner W, Tecle I, Thomason J, Tung CW, Wei X, Yap I, Youens-Clark K, Ware D, Stein L','http://nar.oxfordjournals.org/cgi/content/abstract/36/suppl_1/D947?maxtoshow=&HITS=10&hits=10&RESULTFORMAT=&fulltext=gramene&searchid=1&FIRSTINDEX=0&resourcetype=HWCIT','','',0,'','',''),(30,2008,'Panzea: An Update on New Content and Features','Nucleic Acids Res. 2008 Jan;36(Database issue):D1041-3. Epub 2007 Nov 19','18029361','Canaran P, Buckler ES, Glaubitz JC, Stein L, Sun Q, Zhao W, Ware D','http://nar.oxfordjournals.org/cgi/content/full/36/suppl_1/D1041','','',0,'','',''),(31,2008,'Genome-wide analysis for discovery of new rice miRNAs reveals natural antisense miRNAs (nat-miRNAs)','Proc Natl Acad Sci U S A. 2008;105(12):4951-6','18353984','Lu C, Jeong DH, Kulkarni K, Pillay M, Nobuta K, German R, Thatcher SR, Maher D, Zhang L, Ware D, Liu B, Cao X, Meyers BC, Green PJ','','','',0,'','',''),(32,2008,'Construction, alignment and analysis of 12 framework physical maps that represent the 10 genome types of the genus Oryza','Genome Biol. 2008;9(2):R45. Epub 2008 Feb 28','18304353','Kim, H., B. Hurwitz, Y. Yu, K. Collura, N. Gill, P. Sanmiguel, J.C. Mullikin, C. Maher, W. Nelson, M. Wissotski, M. Braidotti, D. Kudrna, J.L. Goigochia, L. Stein, D. Ware, S.A. Jackson, C. Soderlund, and R.A. Wing','PMC2374706','','',0,'','',''),(33,2008,'The Plant Ontology Database: a community resource for plant structure and developmental stages controlled vocabulary and annotations','Nucleic Acids Research 2008 36(Database issue):D449-D454','PMC2238838','Kim H, Hurwitz B, Yu Y, Collura K, Gill N, SanMiguel P, Mullikin JC, Maher C, Nelson W, Wissotski M, Braidotti M, Kudrna D, Goicoechea JL, Stein L, Ware D, Jackson SA, Soderlund C, Wing RA','http://nar.oxfordjournals.org/cgi/content/full/36/suppl_1/D449?maxtoshow=&HITS=10&hits=10&RESULTFORMAT=&fulltext=plant+ontology&searchid=1&FIRSTINDEX=0&resourcetype=HWCIT','','',0,'','',''),(34,2008,'A new method to compute K-mer frequencies and its application to annotate large repetitive plant genomes','BMC Genomics 9: 517','PMC2613927','Kurtz S, Narechania A, Stein JC, Ware D','','','',0,'','',''),(35,2008,'Low-pass shotgun sequencing of the barley genome facilitates rapid identification of genes, conserved non-coding sequences and novel repeats','BMC Genomics 9:518','18976483','Wicker T, Narechania A, Sabot F, Stein J, Vu GT, Graner A, Ware D, Stein N','','','',0,'','',''),(36,2008,'Handbook of Maize:  History and Practice of Genetics and Genomics','In Bennetzen, J.L. and Jake, S.C. (eds.), Springer Science + Business Media LLC, pp. 659-672','','Lawrence CJ, Ware D','','','',0,'','',''),(37,2009,'Species trees from highly incongruent gene trees in rice','Systematic Biology 2009 58(5):489-500; doi:10.1093/sysbio/syp054','','Cranston K, Hurwitz B, Ware D, Stein L, Wing R','http://sysbio.oxfordjournals.org/cgi/content/short/58/5/489','','',0,'','',''),(38,2009,'A single molecule scaffold for the maize genome','PLoS Genet. 2009 Nov;5(11):e1000711','19936062','Zhou S, Wei F, Nguyen J, Bechner M, Potamousis K, Goldstein S, Pape L, Mehan MR, Churas C, Pasternak S, Forrest DK, Wise R, Ware D, Wing RA, Waterman MS, Livny M, Schwartz DC','http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1000711','','',0,'','',''),(39,2009,'The physical and genetic framework of the maize b73 genome','PLoS Genet. 2009 Nov;5(11):e1000715. Epub 2009 Nov 20','19936061','Wei F, Zhang J, Zhou S, He R, Schaeffer M, Collura K, Kudrna D, Faga BP, Wissotski M, Golser W, Rock SM, Graves TA, Fulton RS, Coe E, Schnable PS, Schwartz DC, Ware D, Clifton SW, Wilson RK, Wing RA','http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1000715','','',0,'','',''),(40,2009,'A genome-wide characterization of microRNA genes in maize','PLoS Genet. 2009 Nov;5(11):e1000716. Epub 2009 Nov 20','19936050','Zhang L, Chia JM, Kumari S, Stein JC, Liu Z, Narechania A, Maher CA, Guill K, McMullen MD, Ware D','http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1000716','','',0,'','',''),(41,2009,'Detailed analysis of a contiguous 22-Mb region of the maize genome','PLoS Genet. 2009 Nov;5(11):e1000728','19936048','Wei F, Stein JC, Liang C, Zhang J, Fulton RS, Baucom RS, De Paoli E, Zhou S, Yang L, Han Y, Pasternak S, Narechania A, Zhang L, Yeh CT, Ying K, Nagel DH, Collura K, Kudrna D, Currie J, Lin J, Kim H, Angelova A, Scara G, Wissotski M, Golser W, Courtney L, Kruchowski S, Graves TA, Rock SM, Adams S, Fulton LA, Fronick C, Courtney W, Kramer M, Spiegel L, Nascimento L, Kalyanaraman A, Chaparro C, Deragon JM, Miguel PS, Jiang N, Wessler SR, Green PJ, Yu Y, Schwartz DC, Meyers BC, Bennetzen JL, Martienssen RA, McCombie WR, Aluru S, Clifton SW, Schnable PS, Ware D, Wilson RK, Wing RA','http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1000728','','',0,'','',''),(42,2009,'A First-Generation Haplotype Map of Maize','Science. 2009 Nov 20;326(5956):1115-7','19965431','Gore MA, Chia JM, Elshire RJ, Sun Q, Ersoz ES, Hurwitz BL, Peiffer JA, McMullen MD, Grills GS, Ross-Ibarra J, Ware DH, Buckler ES','http://www.sciencemag.org/cgi/content/abstract/sci;326/5956/1115','','',0,'','',''),(43,2009,'The B73 maize genome: complexity, diversity, and dynamics','Science. 2009;326(5956):1112-5 doi: 10.1126/science.1178534','19965430','Schnable PS, Ware D, Fulton RS, Stein JC, Wei F, Pasternak S, Liang C, Zhang J, Fulton L, Graves TA, Minx P, Reily AD, Courtney L, Kruchowski SS, Tomlinson C, Strong C, Delehaunty K, Fronick C, Courtney B, Rock SM, Belter E, Du F, Kim K, Abbott RM, Cotton M, Levy A, Marchetto P, Ochoa K, Jackson SM, Gillam B, Chen W, Yan L, Higginbotham J, Cardenas M, Waligorski J, Applebaum E, Phelps L, Falcone J, Kanchi K, Thane T, Scimone A, Thane N, Henke J, Wang T, Ruppert J, Shah N, Rotter K, Hodges J, Ingenthron E, Cordes M, Kohlberg S, Sgro J, Delgado B, Mead K, Chinwalla A, Leonard S, Crouse K, Collura K, Kudrna D, Currie J, He R, Angelova A, Rajasekar S, Mueller T, Lomeli R, Scara G, Ko A, Delaney K, Wissotski M, Lopez G, Campos D, Braidotti M, Ashley E, Golser W, Kim H, Lee S, Lin J, Dujmic Z, Kim W, Talag J, Zuccolo A, Fan C, Sebastian A, Kramer M, Spiegel L, Nascimento L, Zutavern T, Miller B, Ambroise C, Muller S, Spooner W, Narechania A, Ren L, Wei S, Kumari S, Faga B, Levy MJ, McMahan L, Van Buren P, Vaughn MW, Ying K, Yeh CT, Emrich SJ, Jia Y, Kalyanaraman A, Hsia AP, Barbazuk WB, Baucom RS, Brutnell TP, Carpita NC, Chaparro C, Chia JM, Deragon JM, Estill JC, Fu Y, Jeddeloh JA, Han Y, Lee H, Li P, Lisch DR, Liu S, Liu Z, Nagel DH, McCann MC, SanMiguel P, Myers AM, Nettleton D, Nguyen J, Penning BW, Ponnala L, Schneider KL, Schwartz DC, Sharma A, Soderlund C, Springer NM, Sun Q, Wang H, Waterman M, Westerman R, Wolfgruber TK, Yang L, Yu Y, Zhang L, Zhou S, Zhu Q, Bennetzen JL, Dawe RK, Jiang J, Jiang N, Presting GG, Wessler SR, Aluru S, Martienssen RA, Clifton SW, McCombie WR, Wing RA, Wilson RK','http://www.sciencemag.org/cgi/content/abstract/sci;326/5956/1112','','',0,'assets/img/covers/maize.png','',''),(44,2009,'Genetic properties of the maize nested association mapping population','Science. 2009 Aug 7;325(5941):688-9','19661427','McMullen MD, Kresovich S, Villeda HS, Bradbury P, Li H, Sun Q, Flint-Garcia S, Thornsberry J, Acharya C, Bottoms C, Brown P, Browne C, Eller M, Guill K, Harjes C, Kroon D, Lepak N, Mitchell SE, Peterson B, Pressoir G, Romero S, Oropeza Rosas M, Salvo S, Yates H, Hanson M, Jones E, Smith S, Glaubitz JC, Goodman M, Ware D, Holland JB, Buckler ES','','','',0,'','',''),(45,2009,'The genetic architecture of maize flowering time','Science. 2009 Aug 7;325(5941):714-8','19661422','Buckler ES, Holland JB, Bradbury PJ, Acharya CB, Brown PJ, Browne C, Ersoz E, Flint-Garcia S, Garcia A, Glaubitz JC, Goodman MM, Harjes C, Guill K, Kroon DE, Larsson S, Lepak NK, Li H, Mitchell SE, Pressoir G, Peiffer JA, Rosas MO, Rocheford TR, Romay MC, Romero S, Salvo S, Sanchez Villeda H, da Silva HS, Sun Q, Tian F, Upadyayula N, Ware D, Yates H, Yu J, Zhang Z, Kresovich S, McMullen MD','','','',0,'','',''),(46,2009,'CMap 1.01: A comparative mapping application for the Internet','Bioinformatics, doi:10.1093/bioinformatics/btp458','19648141','Youens-Clark K, Faga B, Yap I, Stein L, Ware D','http://bioinformatics.oxfordjournals.org/cgi/content/abstract/btp458?ijkey=RH2dYR27nTL4Bw9&keytype=ref','','',0,'46.jpg','46.pdf','46.doc'),(47,2009,'Large-Scale Discovery of Gene-Enriched SNPs','The Plant Genome doi:10.3835/plantgenome2009.01.0002, vol. 2, no. 2 121-133','','Gore M, Wright M, Ersoz E, Bouffard P, Szekeres E, Jarvie T, Hurwitz B, Narechania A, Harkins T, Grills G, Ware D, Buckler E','http://plantgenome.scijournals.org/content/2/2/121','','',0,'','',''),(48,2009,'Gramene QTL database: development, content and applications','Database: The Journal of Biological Databases and Curation Vol. 2009:bap005; doi:10.1093/database/bap005','20157478','Ni J, Pujar A, Youens-Clark K, Yap I, Jaiswal P, Tecle I, Tung CW, Ren L, Spooner W, Wei X, Avraham S, Ware D, Stein L, McCouch S','http://database.oxfordjournals.org/cgi/content/full/bap005?ijkey=f2bds3udr4Uod87&keytype=ref','','',0,'','',''),(49,2009,'Microdissection of Shoot Meristem Functional Domains','PLoS Genetics 5(5): e1000476. doi:10.1371/journal.pgen.1000476','','Brooks L, Strable J, Zhang X, Ohtsu K, Zhou R, Sarkar A, Hargreaves S, Elshire R, Eudy D, Pawlowska T, Ware D, Janick-Buckner D, Buckner B, Timmermans M, Schnable P, Nettleton D, Scanlon M','http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1000476','','',0,'','',''),(50,2009,'The Sorghum bicolor genome and the diversification of grasses','Nature. 2009 Jan 29;457(7229):551-6','19189423','Paterson AH, Bowers JE, Bruggmann R, Dubchak I, Grimwood J, Gundlach H, Haberer G, Hellsten U, Mitros T, Poliakov A, Schmutz J, Spannagl M, Tang H, Wang X, Wicker T, Bharti AK, Chapman J, Feltus FA, Gowik U, Grigoriev IV, Lyons E, Maher CA, Martis M, Narechania A, Otillar RP, Penning BW, Salamov AA, Wang Y, Zhang L, Carpita NC, Freeling M, Gingle AR, Hash CT, Keller B, Klein P, Kresovich S, McCann MC, Ming R, Peterson DG, Mehboob-ur-Rahman, Ware D, Westhoff P, Mayer KF, Messing J, Rokhsar DS','http://www.nature.com/nature/journal/v457/n7229/abs/nature07723.html','','',0,'','',''),(51,2009,'Evidence-based gene predictions in plant genomes','Genome Res. 2009 Oct;19(10):1912-23','19541913','Liang C, Mao L, Ware D, Stein L','','','',0,'','',''),(52,2010,'An international bioinformatics infrastructure to underpin the Arabidopsis community','Plant Cell. 2010;22(8):2530-6','20807877','International Arabidopsis Informatics Consortium','','','',0,'','',''),(53,2010,'Pervasive gene content variation and copy number variation in maize and its undomesticated progenitor','Genome Res. 2010 Dec;20(12):1689-99','21036921','Swanson-Wagner RA, Eichten SR, Kumari S, Tiffin P, Stein JC, Ware D, Springer NM','','','',0,'','',''),(54,2010,'Gramene database in 2010: updates and extensions','Nucleic Acids Res. 2010 Nov 13','21076153','Youens-Clark K, Buckler E, Casstevens T, Chen C, Declerck G, Derwent P, Dharmawardhana P, Jaiswal P, Kersey P, Karthikeyan AS, Lu J, McCouch SR, Ren L, Spooner W, Stein JC, Thomason J, Wei S, Ware D','','','',0,'','',''),(55,2010,'Digital gene expression signatures for maize development','Plant Physiol. 2010;154(3):1024-39','PMC2971585','Eveland A, Satoh-Nagasawa N, Goldshmidt A, Meyer S, Beatty M, Sakai H, Ware D, Jackson D','','','',0,'','',''),(56,2010,'Phylogenomic analysis from deep BAC-end sequence libraries of rice','Systematic Botany (In press)','','Cranston KA, Hurwitz B, Sanderson MJ, Ware D, Wing R, Stein L','','','',0,'','',''),(57,2010,'Applications and methods utilizing the Simple Semantic Web Architecture and Protocol (SSWAP) for bioinformatics resource discovery and disparate data and service integration.','BioData Min. 2010 Jun 4;3(1):3','20525377','Nelson RT, Avraham S, Shoemaker RC, May GD, Ware D, Gessler DD','','','',0,'','',''),(58,2010,'Rice structural variation: a comparative analysis of structural variation between rice and three of its closest relatives in the genus Oryza','The Plant Journal doi: 10.1111/j.1365-313X.2010.04293.x','20626650','Hurwitz B, Kudrna D, Yu Y, Sebastian A, Zuccolo A, Jackson SA, Ware D, Wing R, Stein L','http://onlinelibrary.wiley.com/doi/10.1111/j.1365-313X.2010.04293.x/abstract','','',0,'','',''),(59,2010,'Rapid genomic characterization of the genus vitis','PLoS One. 2010 Jan 13;5(1):e8219','20084295','Myles S, Chia JM, Hurwitz B, Simon C, Zhong GY, Buckler E, Ware D','http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0008219','ftp://ftp.gramene.org/pub/vitis_plosone_2009_snps/','',0,'','',''),(60,2011,'Functional annotation of the transcriptome of Sorghum bicolor in response to osmotic stress and abscisic acid','BMC Genomics. 2011 Oct 18;12:514','22008187','Dugas DV, Monaco MK, Olson A, Klein RR, Kumari S, Ware D, Klein PE','','','115 on file with USDA Log number 000268556',0,'','',''),(61,2011,'A Transcription Factor Resource and Enhanced Y1H assays to elucidate Arabidopsis Gene Regulatory Networks','Nat Methods. 2011 Oct 30;8(12):1053-5. doi: 10.1038/nmeth.1750','22037706','Gaudinier A, Zhang L, Reece-Hoyes JS, Taylor-Teeples M, Pu L, Liu Z, Breton G, Pruneda-Paz JL, Kim D, Kay SA, Walhout AJ, Ware D, Brady SM','','','115 on file with USDA Log number 000284614',0,'','',''),(62,2011,'BioMart Central Portal: An Open Database Network for Biological Community','Database doi: 10.1093/database/bar041','21930507','Guberman JM, Ai J, Arnaiz O, Baran J, Blake A, Baldock R, Chelala C, Croft D, Cros A, Cutts RJ, Di Genova A, Forbes S, Fujisawa T, Gadaleta E, Goodstein DM, Gundem G, Haggarty B, Haider S, Hall M, Harris T, Haw R, Hu S, Hubbard S, Hsu J, Iyer V, Jones P, Katayama T, Kinsella R, Kong L, Lawson D, Liang Y, Lopez-Bigas N, Luo J, Lush M, Mason J, Moreews F, Ndegwa N, Oakley D, Perez-Llamas C, Primig M, Rivkin E, Rosanoff S, Shepherd R, Simon R, Skarnes B, Smedley D, Sperling L, Spooner W, Stevenson P, Stone K, Teague J, Wang J, Wang J, Whitty B, Wong DT, Wong-Erasmus M, Yao L, Youens-Clark K, Yung C, Zhang J, Kasprzyk A','http://database.oxfordjournals.org/content/2011/bar041.full','','115 submitted to Karen on 6/6/2011',0,'','',''),(63,2011,'GrameneMart: The BioMart Data Portal for the Gramene Project','Database, Vol. 2011, Article ID bar056, doi:10.1093/database/bar056','','William S, Youens-Clark K, Staines D, Ware D','','','',0,'','',''),(64,2011,'The iPlant Collaborative: Cyberinfrastructure for Plant Biology','Frontiers 2011','','Goff S, Vaughn M, Lyons, E, et al., Ware D','','','115 on file with USDA Log number 000284709',0,'','',''),(65,2011,'Evidence for Network Evolution in an Arabidopsis Interactome Map','Science. 2011 Jul 29;333(6042):601-7','21798944','Arabidopsis Interactome Mapping Consortium','','','115 on file with USDA Log number 000269275',0,'','',''),(66,2011,'Sequencing for the cream of the crop','Nat Biotechnol. 2011 Feb;29(2):138-9','21301440','Chia JM, Ware D','','','115 on file with USDA Log number 000284707',0,'','',''),(67,2011,'A stele-enriched gene regulatory network in the Arabidopsis root','Mol Syst Biol. 2011 Jan 18; 7:459','21245844','Brady SM, Zhang L, Megraw M, Martinez NJ, Jiang E, Yi CS, Liu W, Zeng A, Taylor-Teeples M, Kim D, Ahnert S, Ohler U, Ware D, Walhout AJ, Benfey PN','','','115 on file with USDA Log number 000257739',0,'','',''),(68,2011,'Genetic structure and domestication history of the grape','Proc Natl Acad Sci U S A. 2011 Jan 18','21245334','Myles S, Boyko AR, Owens CL, Brown PJ, Grassi F, Aradhya MK, Prins B, Reynolds A, Chia JM, Ware D, Bustamante CD, Buckler ES','','','115 on file with USDA Log number 000260654',0,'','',''),(69,2011,'Genome-wide association study of quantitative resistance to southern leaf blight in the maize nested association mapping population','Nat Genet. 2011 Jan 9','21217757','Kump KL, Bradbury PJ, Wisser RJ, Buckler ES, Belcher AR, Oropeza-Rosas MA, Zwonitzer JC, Kresovich S, McMullen MD, Ware D, Balint-Kurti PJ, Holland JB','','','115 on file with USDA Log number 000256384',0,'','',''),(70,2012,'The tomato genome sequence provides insights into fleshy fruit evolution','Nature 485, 635-641 (31 May 2012)','','Tomato Genome Sequencing Consortium. ','http://www.nature.com/nature/journal/v485/n7400/full/nature11119.html','','115 on file with USDA Log number 000281529',0,'assets/img/covers/tomato.png','',''),(71,2012,'Comparative population genomics of maize domestication and improvement','Nat Genet. 2012 Jun 3. doi: 10.1038/ng.2309','','Hufford M, Xu X, van Heerwaarden J , Pyhajarvi T, Chia JM , Cartwright R, Elshire R, Glaubitz JC, Guill K, Kaeppler S, Lai J , Morrell P, Shannon L, Song C , Springer N, Swanson-Wagner R, Tiffin P, Wang J, Zhang G, Doebley J, McMullen  MD, Ware D, Buckler ES , Yang S, Ross-Ibarra J','','','115 on file with USDA Log number 000271352',0,'','',''),(72,2012,'Maize HapMap2 identifies extant variation from a genome in flux','Nat Genet. 2012 Jun 3. doi: 10.1038/ng.2313.','','Chia JM*, Song C *, Bradbury PJ, Costich D, de Leon N, Doebley J, Elshire RJ, Gaut B, Geller L, Glaubitz JC, Gore M, Guill KE, Holland J, Hufford MB,  Lai J, Li M, Liu X, Lu Y, McCombie WR, Nelson R, Poland J, Prasanna BM, Pyhajarvi T, Rong T, Sekhon RS, Sun Q, Tenaillon MI , Tian F, Wang J, Xu X, Zhang Z, Kaeppler SM, Ross-Ibarra J, McMullen MD, Buckler ES, Zhang G, Xu Y, Ware D ','','','115 on file with USDA Log number 000270704',0,'assets/img/covers/hapmapv2.png','',''),(73,2012,'ZmCCT and the genetic basis of day-length adaptation underlying the postdomestication spread of maize','PNAS June 18, 2012 doi: 10.1073/pnas.1203189109','22711828','Hung HY, Shannon LM, Tian F, Bradbury PJ, Chen C, Flint-Garcia SA, McMullen MD, Ware D, Buckler ES, Doebley JF, Holland JB','','','115 on file with USDA Log number 000279045',0,'','22711828.pdf',''),(74,2012,'Characterization of miRNAs in Response to Short-Term Waterlogging in Three Inbred Lines of Zea mays','PLoS One. 2012;7(6):e39786. Epub 2012 Jun 29','22768123','Liu Z, Kumari S, Zhang L, Zheng Y, Ware D','','','115 on file with USDA Log number 000284611',0,'','',''),(75,2012,'Taking the Next Step: Building an Arabidopsis Information Portal','Plant Cell. 2012 Jun 29','22751211','The International Arabidopsis Informatics Consortium','','','WHO NEEDS TO SUBMIT THE 115 FOR THIS?',0,'','22751211.pdf',''),(76,2012,'GrameneMart: the BioMart data portal for the Gramene project','Database (Oxford). 2012 Feb 28;2012:bar056','22374386','Spooner W, Youens-Clark K, Staines D, Ware D','','','115 on file with USDA Log number 000285216',0,'','',''),(77,2012,'PICARA, an analytical pipeline providing probabilistic inference about a priori candidates genes underlying genome-wide association QTL in plants.','PLoS One. 2012;7(11):e46596','23144785','Chen C, DeClerck G, Tian F, Spooner W, McCouch S, Buckler E','','','115 not required from DW?',0,'','',''),(78,2012,'Maize Metabolic Network Construction and Transcriptome Analysis','The Plant Genome','','Monaco MK, Sen TZ, Dharmawardhana PD, Ren L, Schaeffer M, Naithani S, Amarasinghe V, Thomason1 J, Harper L, Gardiner J, Cannon EKS, Lawrence CJ, Ware D, and Jaiswal P','','','115 on file with USDA Log number 000272324',0,'','',''),(79,2013,'The taxonomic name resolution service: an online tool for automated standardization of plant names','BMC Bioinformatics.2013, 14:16.','23324024','Boyle B, Hopkins N, Lu Z, Raygoza Garay JA, Mozzherin D, Rees T, Matasci N, Narro ML, Piel WH, McKay SJ, Lowry S, Freeland C, Peet RK, Enquist BJ','http://www.biomedcentral.com/1471-2105/14/16','','115 submitted to Karen Barry on 5/17/2013(however Ware not an author so will not be processed through USDA)',0,'','',''),(80,2013,'A Genome Scale Metabolic Network for Rice and Accompanying Analysis of Tryptophan, Auxin and Serotonin Biosynthesis Regulation Under Biotic Stress','Rice','','Dharmawardhana P, Ren L, Amarasinghe V, Monaco M, Thomason J, Ravenscroft D, McCouch S, Ware D and Jaiswal P','','','',0,'','',''),(81,2013,'The maize methylome modulates mRNA splicing and reveals widespread paramutation guided by small RNA','submitted to Genome Research','','Lu, Z., Regulski M, Kendall J, Reinders J, Llaca V, Deschamps S,  Smith A, Levy D, McCombie WR, Tingey S, Rafalski A, Hicks J, Ware D, Martienssen R','','','',1,'','',''),(82,2013,'Genome-wide Computational Prediction and Analysis of Core Promoter Elements Across Plant Monocots and Dicots','submitted to PLOS ONE','','Kumari S and Ware D','','','Yes',1,'','',''),(83,2013,'A 4-gigabase physical map unlocks the structure and evolution of the complex genome of Aegilops tauschii, the wheat D-genome progenitor','Proc Natl Acad Sci U S A. 2013 May 7;110(19):7940-5. doi: 10.1073/pnas.1219082110. Epub 2013 Apr 22.','23610408','Luo MC, Gu YQ, You FM, Deal KR, Ma Y, Hu Y, Huo N, Wang Y, Wang J, Chen S, Jorgensen CM, Zhang Y, McGuire PE, Pasternak S, Stein JC, Ware D, Kramer M, McCombie WR, Kianian SF, Martis MM, Mayer KF, Sehgal SK, Li W, Gill BS, Bevan MW, Simkova H, Dolezel J, Weining S, Lazo GR, Anderson OD, Dvorak J.','http://www.ncbi.nlm.nih.gov/pubmed/23610408','','Yes',0,'','','');
/*!40000 ALTER TABLE `pub` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Dumping data for table `pub_to_funding`
--

LOCK TABLES `pub_to_funding` WRITE;
/*!40000 ALTER TABLE `pub_to_funding` DISABLE KEYS */;
INSERT INTO `pub_to_funding` VALUES (1,1,1),(2,2,1),(3,3,1),(4,4,1),(5,5,1),(6,6,1),(7,7,1),(8,8,1),(9,9,1),(10,10,1),(11,11,1),(12,12,1),(13,13,1),(14,14,1),(15,15,1),(16,16,1),(17,17,1),(18,18,1),(19,19,1),(20,20,1),(21,21,1),(22,22,1),(23,23,1),(24,24,1),(25,25,1),(26,26,1),(27,27,1),(28,28,1),(29,29,1),(30,30,1),(31,31,1),(32,32,1),(33,33,1),(34,34,1),(35,35,1),(36,36,1),(37,37,1),(38,38,1),(39,39,1),(40,40,1),(41,41,1),(42,42,1),(43,43,1),(44,44,1),(45,45,1),(46,46,1),(128,46,2),(47,47,1),(48,48,1),(49,49,1),(50,50,1),(51,51,1),(52,52,1),(53,53,1),(54,54,1),(55,55,1),(56,56,1),(57,57,1),(58,58,1),(59,59,1),(60,60,1),(61,61,1),(62,62,1),(63,63,1),(64,64,1),(65,65,1),(66,66,1),(67,67,1),(68,68,1),(69,69,1),(70,70,1),(71,71,1),(72,72,1),(73,73,1),(74,74,1),(75,75,1),(76,76,1),(77,77,1),(78,78,1),(79,79,1),(80,80,1),(81,81,1),(82,82,1),(83,83,1);
/*!40000 ALTER TABLE `pub_to_funding` ENABLE KEYS */;
UNLOCK TABLES;
/*!40103 SET TIME_ZONE=@OLD_TIME_ZONE */;

/*!40101 SET SQL_MODE=@OLD_SQL_MODE */;
/*!40014 SET FOREIGN_KEY_CHECKS=@OLD_FOREIGN_KEY_CHECKS */;
/*!40014 SET UNIQUE_CHECKS=@OLD_UNIQUE_CHECKS */;
/*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
/*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
/*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
/*!40111 SET SQL_NOTES=@OLD_SQL_NOTES */;

-- Dump completed on 2013-09-11 21:22:40
